src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -676,11 +676,14 @@ private void DeleteFile(string path)
             }
         }
 
-        private void ValidateFreeSpaceForSnapshot(string filePath)
+        private void ValidateFreeSpaceForSnapshot(string filePath, bool excludeIndexes)
         {
             long totalUsedSpace = 0;
             foreach (var mountPointUsage in _database.GetMountPointsUsage(includeTempBuffers: false))
             {
+                if(mountPointUsage.Type == "Index" && excludeIndexes)
```

Comment:
You can use `StorageEnvironmentType` here

Reply:
Done.